src/utils/getSiblings.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Get all siblings before passed element, or after it
+ */
+export function getSiblings(element: HTMLElement, afterCurrentElement: boolean = true): Element[] | null {
```

Comment:
lets make it more clear 
```suggestion
export function getSiblings(element: HTMLElement, direction: 'after' | 'before'): Element[] | null {
```